docker-hub/access-tokens.md

```diff
@@ -7,62 +7,83 @@ keywords: docker hub, hub, security, PAT, personal access token
+
 
 ## Create an access token
 
 You can create as many tokens as you need.
 
-1. Log in to [hub.docker.com](https://hub.docker.com).
-
-2. Click on your username in the top right corner and select **Account
-Settings**.
+1. Log in to [hub.docker.com](https://hub.docker.com){: target="_blank" rel="noopener" class="_"}.
 
-3. Select **Security > New Access Token**.
+2. Click on your username in the top right corner and select **[Account Settings](https://hub.docker.com/settings/general)**.
 
-      ![](images/hub-create-token.png)
+3. Select **[Security](https://hub.docker.com/settings/security) > New Access Token**.
 
 4. Add a description for your token. Use something that indicates where
-the token is going to be used, or set a purpose for the token.
+the token will be used, or set a purpose for the token.
+
+      ![PAT Menu](images/hub-create-token.png){:width="700px"}
+
+5. You can view the following permissions from the drop-down:
 
-5. Copy the token that appears on the screen. Make sure you do this now:
-once you close this prompt, Docker will never show the token again.
+    | Features   | Free | Pro | Team |
+    | --------------------- | ---- | ----- | ----- |
+    | Personal Access Tokens (read, write, delete scope only)*    |  x |   x  |   x  |
+    | Scoped Personal Access Tokens  |      | x   |   x   |
+    | Unlimited Personal Access Tokens   |      |       |   x   |
 
-      ![](images/hub-copy-token.png)
+      >*You can create 1 access token as part of the Free plan, 5 tokens per the Pro plan, and Unlimited tokens for the Team plan.
+      For more information about a Pro or a Team plan, see [Docker Pricing](https://www.docker.com/pricing?utm_source=docker&utm_medium=webreferral&utm_campaign=docs_driven_upgrade){: target="_blank" rel="noopener" class="_"}.
 
-      Treat access tokens like your password and keep them secret. Store your tokens securely (for example, in a credential manager).
+6. Copy the token that appears on the screen and save it. You will not be able to retrieve the token once you close this prompt.
 
+      ![Copy Token](images/hub-copy-token.png){:width="700px"}
+
+## Use an access token
+
+You can use an access token anywhere that requires your Docker Hub
+password.
+
+When logging in from your Docker CLI client (`docker login --username <username>`), omit the password in the login command. Instead, enter your token when asked for a password.
+
+   > **Note**
+   >
+   >If you have 2FA enabled, you must use a personal access token when logging in from the Docker CLI. 2FA is an optional, but more secure method of authentication.
 
 ## Modify existing tokens
 
-You can rename, deactivate, or delete a token as needed.
+You can rename, activate, deactivate, or delete a token as needed.
 
-1. Access your tokens under **Account Settings > Security**.
+1. Access your tokens under **[Account Settings](https://hub-stage.docker.com/settings/general) > [Security](https://hub-stage.docker.com/settings/security)**.
```

Comment:
Looks like these still point to the staging domain;

```suggestion
1. Access your tokens under **[Account Settings](https://hub.docker.com/settings/general) > [Security](https://hub.docker.com/settings/security)**.
```

Do we want these to open in a new window?

```suggestion
1. Access your tokens under **[Account Settings](https://hub.docker.com/settings/general){: target="_blank" rel="noopener" class="_"} > [Security](https://hub.docker.com/settings/security){: target="_blank" rel="noopener" class="_"}**.
```

Reply:
Wah, sorry for the repeated comments. Looking at this on my machine, and it looks like we want to point users to the "security" tab here, so we may not need the first link ("general"), so this could be a single link with `Account Settings > Security` as caption;

```suggestion
1. Access your tokens under **[Account Settings > Security](https://hub.docker.com/settings/security){: target="_blank" rel="noopener" class="_"}**.
```